tests/scene/test_node.h

```diff
@@ -146,12 +146,12 @@ TEST_CASE("[SceneTree][Node] Testing node operations with a very simple scene tr
 	}
 
 	SUBCASE("Node should be possible to find") {
-		Node *child = SceneTree::get_singleton()->get_root()->find_child("Node", true, false);
+		Node *child = SceneTree::get_singleton()->get_root()->find_child("Node", true, false, "");
```

Comment:
```suggestion
		Node *child = SceneTree::get_singleton()->get_root()->find_child("Node", true, false);
```
All these should be removed

Reply:
I have made the requested change.